### PR TITLE
created parser

### PR DESCRIPTION
**Pull request by tomcorey26**

### CODE REVIEW

**Comment by tomcorey26** [comment]
rob add hacktoberfest tag so i get credit for this

File: parser.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import csv
```
Comment (RobertCorey) [inline parser.py:1]: are these native node packaged?

Comment (RobertCorey) [reply parser.py:1]: @tomcorey26

**Review by RobertCorey — APPROVED** [review]

File: parser.py (hunk 1 above)
Comment (RobertCorey) [reply parser.py:1]: oh nevermind i see it is python

**Pull request merged**